src/z3c/etestbrowser/browser.py

```diff
@@ -91,24 +88,11 @@ def etree(self):
     def normalized_contents(self):
         if self._normalized_contents is None:
             indent(self.etree)
-            self._normalized_contents = lxml.etree.tostring(
+            self._normalized_contents = lxml.etree.tounicode(
                 self.etree, pretty_print=True)
```

Comment:
lxml's tounicode() is [deprecated](https://lxml.de/3.1/api/lxml.etree-module.html#tounicode):

> Deprecated: use tostring(el, encoding=unicode) instead.

(The docs for tostring() clarify that you can use encoding='unicode', as well as encoding=six.text_type.)

Reply:
Thank you for digging this up.

Reply:
It was a pure accident: I was trying to see if there's a way to wrap long attribute lists when pretty-printing them with lxml.etree.tostring() and was therefore browsing lxml's docs the day before.